unittests.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import time, datetime
+
+import project03
+import examples
+
+class TestMarriageMethods(unittest.TestCase):
+
+    def testTimestamp(self):
+        self.assertEqual(project03.gedcomDateToUnixTimestamp('01 JAN 1970'), time.mktime(datetime.datetime.strptime('1/1/1970', '%d/%m/%Y').timetuple()))
+        self.assertTrue(project03.gedcomDateToUnixTimestamp('28 SEP 2020') == project03.gedcomDateToUnixTimestamp('28 SEP 2020'))
+        self.assertTrue(project03.gedcomDateToUnixTimestamp('28 SEP 2020') > project03.gedcomDateToUnixTimestamp('28 SEP 2019'))
+
+    def testMarriageBeforeDivorceSuccess(self):
+        self.assertTrue(project03.verifyMarriageBeforeDivorce(examples.exampleFamilyDivorced))

```

Comment:
If we update the way our dicts work internally we also have to update how examples work which sucks. We should have a blackbox (tested) function that parses gedcom into the dict and have one or more example gedcom files that we can parse into dicts and then pass in. Refactor work for next sprint?

Reply:
Sounds good